stable/appmesh-gateway/templates/gateway.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.appmesh.gateway }}
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: {{ template "appmesh-gateway.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "appmesh-gateway.labels" . | indent 4 }}
+spec:
+  namespaceSelector:
+    matchLabels:
+      appmesh.k8s.aws/sidecarInjectorWebhook: enabled
```

Comment:
Is there a reason for using pod injection label as virtual gateway namespaceSelector? It'd be better to make to use an explicit label instead to avoid selecting all the namespaces across all the meshes if customers use `appmesh.k8s.aws/sidecarInjectorWebhook: enabled`

Reply:
I think that's an advanced use-case, the chart tries to make it easy for those that are running a single ingress. I would expect users that what to run multiple gateways for multiple meshes to set `appmesh.gateway=false` and create their own objects.

Reply:
Sounds good